installing/installing_alibaba/manually-creating-alibaba-ram.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: ASSEMBLY
+[id="manually-creating-alibaba-ram"]
+= Manually manage cloud credentials
+include::modules/common-attributes.adoc[]
+:context: manually-creating-alibaba-ram
+
+toc::[]
+
+// https://github.com/openshift/cloud-credential-operator/pull/412/files#diff-2480a11ca4927139d6eaa9883946b6f4cb38358cd98def8c57dd73e9319dbc9cR232
+
+For the Alibaba Cloud,  you can use the CCO utility (`ccoctl`) to create credential requests for the {product-title} installer. You can also use `ccoctl` to create a Resource Access Management (RAM) user with a long-lived AccessKey (AK) pair for each {product-title} component. Every RAM user who owns the AK is configured with RAM policies that match the permission defined in each credentials request.
+
+For more information about CCO credential modes, see xref:../../authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc[About the Cloud Credential Operator].
+
```

Comment:
This module will need a few edits to make it not AWS-specific, but I can handle that in a separate PR.

```suggestion

//Task part 1: Configuring the Cloud Credential Operator utility
include::modules/cco-ccoctl-configuring.adoc[leveloffset=+1]

```